tightbind/Zmat.c

```diff
@@ -244,7 +244,7 @@ void eval_Zmat_locs(atom_type *atoms,int num_atoms, int cell_dim,char printing)
     /* now print out the positions of the atoms */
     fprintf(output_file,"# Positions of atoms from Z-matrix\n");
     for(i=0;i<num_atoms;i++){
-      fprintf(output_file,"% 4d % 4s % 8.4lf % 8.4lf % 8.4lf\n",
```

Comment:
I believe the space used to be (i.e. in the mid-90s) necessary to get space-padding. Looks like that isn't true any more.